packages/runtime/container-runtime/src/summary/summaryFormats.md

```diff
@@ -0,0 +1,150 @@
+# Summary and snapshot formats
+
+## Introduction
+
+This document describes the format of summaries and snapshots. This should help developers understand how to look at the data, how to make changes, etc. It should also help users understand the format of their data in summary / snapshot.
```

Comment:
Consider rephrasing to something like: Purpose of this document is to briefly explain the contents of summaries or snapshots that the runtime layer generates, and to possibly enable developers to make changes to their summaries/snapshot if ever needed.

Reply:
I rewrote this section to better describe what's in it.